jquery.flot.js

```diff
@@ -1459,14 +1497,12 @@ Licensed under the MIT license.
                 switch(type) {
                     case 'min':
                     case 'max':
-                        // this is a workaround to display the endpoints with a higher
-                        //precision without changing the API and all the tickFormatters
-                        //axis.tickDecimals = (axis.tickDecimals != null ? axis.tickDecimals + 1 : null);
-                        label = axis.tickFormatter(v, axis);
-                        //axis.tickDecimals = (axis.tickDecimals != null ? axis.tickDecimals - 1 : null);
+                        // display endpoints with higer precision
+                        var precision = plot.computeValuePrecision(axis.min, axis.max, axis.direction, axis.options);
```

Comment:
Since we do not call that function with tickDecimals parameter, it will always compute the precision based on the axis.min and axis.max range. This means that the number of decimals will be variated based on axis interval. If we have a small interval, then we will see a number with more decimals (i.e for 1.000 - 1.001, we will have 4 decimals for endpoints, even if the precision used by major ticks will be 3)